### DISCUSSION TITLE
什么是事件循环

### DISCUSSION THREAD

**Discussion by giscus** [ztkuaikuai/blog#3] [General]
# 什么是事件循环

记录一些技术、生活、阅读的感悟；是我的人类使用说明书。

https://blog.kuaikuaitz.top/posts/%E4%BA%8B%E4%BB%B6%E5%BE%AA%E7%8E%AF

**Comment by ztkuaikuai** [comment]
🥺🥺泰酷啦

**Comment by ztkuaikuai** [comment]
just test ：）

**Comment by mileswangs** [comment]
佬，太酷了
